Library/Formula/unrar.rb

```diff
@@ -15,6 +15,11 @@ class Unrar < Formula
   def install
     system "make"
     bin.install 'unrar'
+    system "make", "clean"
```

Comment:
The makefile is a bit odd, it seems to be doing an implicit "clean" in the background when the second build starts.  This isn't an issue in an interactive shell, but in the brew automated environment it confused the dependencies.  The explicit "clean" takes care of the issues with that.  I will add a comment about this in the formula.
(A patch to the makefile would be a bit excessive, and violate KISS, in my opinion.)